.github/workflows/cmake.yml

```diff
@@ -19,6 +19,7 @@ jobs:
             cmake_generator: "Unix Makefiles"
             cmake_options:
               -DOSS=ON
+            werror_flags: "-Werror"
```

Comment:
This should be done in CMakeLists.txt for `${CMAKE_BUILD_TYPE} STREQUAL "Debug"` and change the `cmake_build_type` variable in this GH Actions workflow file to `Debug`.